frontend/app-development/layout/SettingsModalButton/SettingsModal/components/Tabs/AccessControlTab/AccessControlTab.test.tsx

```diff
@@ -11,23 +11,17 @@ import { textMock } from '../../../../../../../testing/mocks/i18nMock';
 import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
 import type { ServicesContextProps } from 'app-shared/contexts/ServicesContext';
 import { ServicesContextProvider } from 'app-shared/contexts/ServicesContext';
-import type { QueryClient, UseMutationResult } from '@tanstack/react-query';
-import userEvent from '@testing-library/user-event';
-import { useAppMetadataMutation } from 'app-development/hooks/mutations';
-import type { ApplicationMetadata } from 'app-shared/types/ApplicationMetadata';
+import type { QueryClient } from '@tanstack/react-query';
 import { mockAppMetadata } from '../../../mocks/applicationMetadataMock';
+import userEvent from '@testing-library/user-event';
 
 const mockApp: string = 'app';
 const mockOrg: string = 'org';
-
-jest.mock('../../../../../../hooks/mutations/useAppMetadataMutation');
 const updateAppMetadataMutation = jest.fn();
-const mockUpdateAppMetadataMutation = useAppMetadataMutation as jest.MockedFunction<
-  typeof useAppMetadataMutation
->;
-mockUpdateAppMetadataMutation.mockReturnValue({
-  mutate: updateAppMetadataMutation,
-} as unknown as UseMutationResult<void, Error, ApplicationMetadata, unknown>);
+
+jest.mock('app-development/hooks/mutations', () => ({
+  useAppMetadataMutation: () => ({ mutate: updateAppMetadataMutation }),
+}));
```

Comment:
You can remove this and pass the mocked query to the render(AndResolve) functions instead:
`{updateAppMetadata: updateAppMetadataMock}`

Reply:
Fixed